### ISSUE TITLE
[tck] OSGi Core TCKs are missing on central

### ISSUE THREAD

**Issue by laeubi** [osgi/osgi#557]
There is: https://github.com/osgi/osgi/tree/main/org.osgi.test.cases.log but I cant find `org.osgi.test.cases.log` on central, was it not deployed as part of the 8.1 release?

**Comment by bjhargrave** [comment]
We just made the Compendium 8.1 release which, for the very first time, includes TCKs in the release. The `org.osgi.test.cases.log` TCK is part of Core. Since we have not made a Core release since the build was updated to enable releasing TCKs to Maven Central, there are no Core TCKs in Maven Central. When we make the next Core release, then we will get the Core TCKs in Maven Central.

**Comment by laeubi** [comment]
can we have a core 8.1. release as well or at least deploy the tck artifacts for 8.0.0? Would be good to keep this open to possibly discuss this in tomorrow specification call!

**Comment by bjhargrave** [comment]
Core R8 has 12 TCK artifacts:

org.osgi.test.cases.bundle.annotations-8.0.0
org.osgi.test.cases.log-8.0.0
org.osgi.test.cases.condpermadmin-8.0.0
org.osgi.test.cases.log.launch-8.0.0
org.osgi.test.cases.framework-8.0.0
org.osgi.test.cases.permissionadmin-8.0.0
org.osgi.test.cases.framework.launch-8.0.0
org.osgi.test.cases.resolver-8.0.0
org.osgi.test.cases.framework.launch.secure-8.0.0
org.osgi.test.cases.tracker-8.0.0
org.osgi.test.cases.framework.secure-8.0.0
org.osgi.test.cases.url-8.0.0

I extracted each jar from the official Core R8 TCK archive: https://github.com/osgi/materials/blob/main/Core%20Release%208%20(EFSP)/repo/org/osgi/osgi.tck.core/8.0.0/osgi.tck.core-8.0.0.jar.

For each jar, I created an empty `-javadoc` jar, extracted the pom file from `META-INF/maven` and the source from `OSGI_OPT/src`. The `-sources` jar was then created with:

`jar cvf $ARTIFACT-sources.jar -C OSGI-OPT/src org`

I then created a staging repo on OSSRH (`orgosgi-1256` in this instance) and deployed each TCK with:

`mvn gpg:sign-and-deploy-file -Dgpg.ascDirectory=asc -DrepositoryId=ossrh -Durl=https://$OSSRH_USERNAME:$OSSRH_PASSWORD@oss.sonatype.org/service/local/staging/deployByRepositoryId/orgosgi-1256 -Dfile=$ARTIFACT.jar -DpomFile=$ARTIFACT.pom -Djavadoc=$ARTIFACT-javadoc.jar -Dsources=$ARTIFACT-sources.jar`

I then closed and released the staging repo.

The above resulted in the official TCK artifacts being deployed to Maven Central. The original build for Core R8 did not deploy TCK artifacts and changing the build to add that support would result in slightly different binaries which would technically have been a service release. Deploying the official TCK artifacts was a pure act of publishing and not a new release.
